recipes/cookbooks/ntp/attributes/default.rb

```diff
@@ -5,6 +5,11 @@
 when "redhat","centos","fedora"
   default[:ntp][:service] = "ntpd"
   default[:ntp][:root_group] = "root"
+  platform_version = node['platform_version'].split('.').first
+  if %w(6 7).include?(platform_version)
+    default[:centos_repo_baseurl] = "http://ftp.heanet.ie/pub/centos/#{platform_version}/os/x86_64/"
```

Comment:
The key clearly states :centos_repo_baseurl. This key is unique and will not be overridden by other options. Please set this attribute outside the case statement.